x-pack/libbeat/management/manager.go

```diff
@@ -96,24 +101,26 @@ func (cm *Manager) Enabled() bool {
 	return cm.config.Enabled
 }
 
-// Start the config manager
-func (cm *Manager) Start(stopFunc func()) {
-	if !cm.Enabled() {
-		return
-	}
-
+// Start the config manager.
+func (cm *Manager) Start() error {
 	cm.lock.Lock()
 	defer cm.lock.Unlock()
 
+	if !cm.Enabled() {
+		return nil
+	}
+
 	cfgwarn.Beta("Fleet management is enabled")
 	cm.logger.Info("Starting fleet management service")
 
-	cm.stopFunc = stopFunc
 	cm.isRunning = true
 	err := cm.client.Start(context.Background())
 	if err != nil {
 		cm.logger.Errorf("failed to start elastic-agent-client: %s", err)
+	} else {
+		cm.logger.Info("Ready to receive configuration")
```

Comment:
```suggestion
		cm.logger.Info("ready to receive configuration")
````

Will merge when other review are in, so I don't have to wait the ci.